react/react1/week3/todo-app-week3/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import Header from './components/Header';
+import Timer from './components/Timer';
+import TodoList from './components/TodoList';
+import Form from './components/Form';
+import './App.css';
+
+function App() {
+  const [input, setInput] = useState('');
+  const [todos, setTodos] = useState([]);
```

Comment:
Only todos and setTodos are state that is being used with multiple components so why not move the input and deadlineValue state to the components where they are used? This way you decrease the amount of props you are passing and you will have a better structure of your components as they are not as closely coupled.